improvelib/utils.py

```diff
@@ -584,39 +579,35 @@ def store_predictions_df(params: Dict,
     return None
 
 
-def compute_performance_scores(params: Dict,
-                               y_true: np.array,
+def compute_performance_scores(y_true: np.array,
                                y_pred: np.array,
-                               stage: str,
-                               outdir: Union[Path, str],
-                               metrics: List):
+                               stage: str, 
```

Comment:
If this is a controlled vocabulary, and used by model developers, then we should create a specific Type for that.

Reply:
LGTM? what are you referring to, stage?